### PR TITLE
Auto-pull-request-on-2021-01-23

### PR DESCRIPTION
**Pull request by Rory-Z**

### CODE REVIEW

File: apps/emqx_auth_pgsql/etc/emqx_auth_pgsql.conf [hunk 1]
```diff
@@ -47,6 +47,14 @@ auth.pgsql.ssl = off
 ##
 ## auth.pgsql.ssl_opts.tls_versions = tlsv1.2
 
+## TLS version
+## You can configure multi-version use "," split,
+## default value is :tlsv1.2
+## Example:
+##    tlsv1.1,tlsv1.2,tlsv1.3
+##
+## auth.pgsql.ssl.tls_versions = tlsv1.2
```
Comment (zmstone) [inline apps/emqx_auth_pgsql/etc/emqx_auth_pgsql.conf:56]: this needs quote.

Comment (zmstone) [reply apps/emqx_auth_pgsql/etc/emqx_auth_pgsql.conf:56]: And a lot other places in this config.
Ping @z8674558 to have a look, If you think it's better to fix them in another PR, please help to merge this one.

**Review by zmstone — APPROVED** [review]
LGTM
except for the configs in `emqx_auth_pgsql.conf` which can be followed up in a separate PR for v5.0

File: apps/emqx_auth_pgsql/etc/emqx_auth_pgsql.conf (hunk 1 above)
Comment (z8674558) [reply apps/emqx_auth_pgsql/etc/emqx_auth_pgsql.conf:56]: you do not have to quote string with `.` .
```
5> hocon:binary("a=aa.bb").
{ok,#{a => <<"aa.bb">>}}
```

**Comment by zmstone** [comment]
force pushed dev/v5.0 to the commit 84e92220

**Comment by zmstone** [comment]
84e92220 is the commit before this squashed merge.

**Pull request merged**